py/greetings.py

```diff
@@ -1,19 +1,36 @@
+import html
 import dominate
 from dominate.tags import *
 
-def _greetings():
-    """Returns the generated HTML greetings. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+def _greetings(week):
```

Comment:
nit: `week` can be a misleading param name since I might just input the number of the week, can we rename into something more descriptive that won't mislead?

Reply:
Renamed